config/checkstyle-non-main-files-checks.xml

```diff
@@ -57,7 +57,7 @@
     <property name="id" value="lineLength"/>
     <!-- catch lines above 100 symbols -->
     <property name="format"
-             value="^(?!(.*href=|.*http(s)?:|import |(.* )?package |.* files=|.*\.dtd| \* \{@code| \* com\.)).{101,}$"/>
+             value="^(?!(.*value=.*Example\d+|.*href=|.*http(s)?:|import |(.* )?package |.* files=|.*\.dtd| \* \{@code| \* com\.)).{101,}$"/>
```

Comment:
The `path` value in macros tends to be long.
```xml
        <macro name="example">
          <param name="path"
                 value="resources/com/puppycrawl/tools/checkstyle/checks/whitespace/filetabcharacter/Example4.xml"/>
          <param name="type" value="code"/>
        </macro>
```

Reply:
Please suppress path as we do for links

Reply:
So like it is right now?

Reply:
Yes